### ISSUE TITLE
text font family gets changed if one tspan is not supported

### ISSUE THREAD

**Issue by isaacfink** [RazrFalcon/resvg#589]
I have this svg code
```
 <text
        filter="url(#textShadow-uy58f0o68)"
        y="1127"
        x="525"
        text-anchor="middle"
        letter-spacing="0.2em"
        dominant-baseline="hanging"
        font-size="30"
        fill="#5a361f"
        transform="rotate(0 525 1178)">
              <tspan font-family="Halimun, Concept, sans-serif">Y</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">a</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">n</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">k</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">y</tspan>
              <tspan font-family="Halimun, Concept, sans-serif"> </tspan>
              <tspan font-family="Halimun, Concept, sans-serif">&amp;</tspan>
              <tspan font-family="Halimun, Concept, sans-serif"> </tspan>
              <tspan font-family="Halimun, Concept, sans-serif"> </tspan>
              <tspan font-family="Halimun, Concept, sans-serif">S</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">h</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">e</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">v</tspan>
              <tspan font-family="Halimun, Concept, sans-serif">y</tspan>      
</text>
```
I am expecting the `tspan` with a missing glyph should be the fallback font family and the rest should be the correct one but that is not the case, instead the entire `text` renders in a different font, is this intended behavior? if yes how can I get this functionality?

For my application I am not in control of the fonts used and also not what text is rendered so to minimize the impact of missing glyphs I was planning on only changing the character that is missing

**Comment by isaacfink** [comment]
when using it through the cli i get the following error
`Warning (in usvg_text_layout:1392): Fallback from Halimun to Arial Unicode MS.`
the fonts are loaded in correctly and I know that because the next line is loaded with the same font and it works, the missing glyph is the `&` and when I remove the `tspan` entirely it the whole text renders correctly

**Comment by isaacfink** [comment]
I created a minimal reproduction in the browser (chromium, safari and firefox) it looks like this
![image](https://user-images.githubusercontent.com/46644190/222181029-e97ca227-a69a-4d4a-9b86-f198b36665ac.png)
and this is the html code to render it
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    
  </head>
  <body>
      <svg
        version="1.1"
        width="100%"
        height="100%"
        xmlns="http://www.w3.org/2000/svg"
        viewBox="0 0 1080 1080"
        background-color="red"
      >
         <text
          y="540"
          x="540"
          text-anchor="middle"
          letter-spacing="0.2em"
          dominant-baseline="hanging"
          font-size="72"
          fill="#5a361f"
          transform="rotate(0 525 1178)">
            <tspan font-family="Halimun, sans-serif">A</tspan>
            <tspan font-family="Halimun, sans-serif">&amp;</tspan>
            <tspan font-family="Halimun, sans-serif">B</tspan>
        </text>
      </svg>
  </body>
  <style>
    @font-face {
      font-family: 'Halimun';
      src: url('./fonts/halimun.ttf') format('truetype');
      font-weight: normal;
      font-style: normal;
    }
  </style>
</html>
```
but when I run this command `./resvg --use-fonts-dir ./fonts in.svg out.png` with just the svg it gives me this
![image](https://user-images.githubusercontent.com/46644190/222182452-6a0f2dff-9454-42e3-8f8b-814ff8ad7ac6.png)
the font is completely different, i am loading the same font file in the html and in the cli command and the name is correct (at least according to typeface app) I am not sure what the the behavior should be according to the spec but it works as intended in all major browsers so it seems like a bug

**Comment by isaacfink** [comment]
this is what I get when I just remove the second `tspan` 
![image](https://user-images.githubusercontent.com/46644190/222184200-cf6d237a-fc79-4d2a-99ff-3fe3d05987b0.png)

**Comment by RazrFalcon** [comment]
This is a known issue. It would be fixed by #486 eventually.
